README.md

```diff
@@ -1,2 +1,93 @@
-# api-webapp
-This project will consume a real API
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [API WebApp] <a name="about-project"></a>
+
+**[Api WebApp]** A movie website built with tvmaze API which shows episodes of a seasonal movie. Users get to share their comments through the comment section in the popup menu.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS, JavaScript with webpack and linter files
+
+### Key Features <a name="key-features"></a>
+
+- **[linter files(debugging tools)]**
+- **[ html, JavaScript and css template]**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Movies API](https://danifromecuador.github.io/api-webapp/dist/)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Install git on your computer
+
+-code editor
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+-git@github.com:Danifromecuador/api-webapp.git
+
+### Deployment
+
+N/A
+
+
+## 👥 Authors <a name="authors"></a>
+
+### Autor 1
+- GitHub: [@Mylo16](https://github.com/Mylo16)
+- Twitter: @EricAnt33872607
+- LinkedIn: linkedin.com/in/eric-antwi-580650251
+
+### Author 2
+- GitHub: [@danifromecuador](https://github.com/danifromecuador)
+- Twitter: [@danifromecuador](https://twitter.com/danifromecuador)
+- LinkedIn: [danifromecuador](https://www.linkedin.com/in/danifromecuador/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- In the near future We will be adding the other sections like watching the movies online, downloading, etc
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+-https://github.com/Danifromecuador/api-webapp/issues
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project show your support by giving a ⭐️
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank the entire microverse staff and students for such a wonderful coding camp
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] It would be better if we create the MIT license file in our root directory so that we can fix the `404` error page not found